packages/lodestar-params/src/presets/mainnet/phase1.ts

```diff
@@ -8,7 +8,7 @@ export const phase1Json = {
   // ---------------------------------------------------------------
   PHASE_1_FORK_VERSION: "0x01000000",
   // [STUB]
-  PHASE_1_FORK_SLOT: 0,
+  PHASE_1_FORK_SLOT: "99999999999",
```

Comment:
why make this a string if the type is a number?  `INITIAL_ACTIVE_SHARDS` (just below this one) is a number and has a number value as well

Reply:
I just copied the value in lightclient.ts, `LIGHTCLIENT_PATCH_FORK_SLOT`
Shouldn't really matter either way, but these objects mirror the yaml config in the specs.
`createIBeaconParams` treats each value as a `Json`, and coerces it to the right type. (this is more important for byte arrays, eg: `0x01000000` -> `Uint8Array[1, 0, 0, 0]`)